src/lazyload-image.directive.ts

```diff
@@ -1,10 +1,10 @@
-import { AfterContentInit, Directive, ElementRef, EventEmitter, Inject, Input, NgZone, OnChanges, OnDestroy, Optional, Output } from '@angular/core';
+import { AfterContentInit, Directive, ElementRef, EventEmitter, Inject, Input, NgZone, OnChanges, OnDestroy, Optional, Output, PLATFORM_ID } from '@angular/core';
+import { isPlatformServer } from "@angular/common";
```

Comment:
Nice, can you change to `'` here?
```suggestion
import { isPlatformServer } from '@angular/common';
```

Reply:
hauahaha, sorry